update.sh

```diff
@@ -63,9 +63,12 @@ if [ "$TAG" == "$TAG_EXISTS" ]; then
     exit 1
 fi
 
+
 if [[ ! -f "$TEMPLATE" ]]; then
     echo "Template $TEMPLATE does not exist"
     exit 1
 fi
 
-sed -e "s/XXX/$CRATE_VERSION/g" -e "s/YYY/$CRASH_VERSION/g"  "$TEMPLATE" > Dockerfile
+BUILD_TIMESTAMP=$(TZ=UTC date '+%FT%T.%N%:z')
+
+sed -e "s/XXX/$CRATE_VERSION/g" -e "s/YYY/$CRASH_VERSION/g" -e "s/AAA/$BUILD_TIMESTAMP/g" "$TEMPLATE" > Dockerfile
```

Comment:
I'd replace this placeholder variables (XXX,...) with a more meaningful value.

Reply:
Agreed. To be honest, I would like to replace this with a proper templating format (jinja?) rather than `sed`ing strings.

Reply:
Like, something along these lines: https://github.com/crate/docker-crate/commit/a466cb2e1bee8ce260435f320cbedbf4476c8090